spec/features/course/assessment/submissions_viewing_spec.rb

```diff
@@ -72,23 +72,17 @@
     context 'As a Course Student' do
       let(:user) { create(:course_student, course: course).user }
 
-      scenario 'I can view my submitted and graded submissions' do
```

Comment:
@fonglh Can you update the status of submission in the middle and `visit current_path` after updating ? so you don't need to remove those 2 statuses from feature spec.

@weiqingtoh What do you think ?